src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -409,6 +409,18 @@ default int getMaxBitrate()
         return Math.max(maxBitrate, getBoostTier().getMaxBitrate());
     }
 
+    /**
+     * The maximum amount of emotes a guild can have.
+     *
+     * @return The maximum amount of emotes
+     *
+     * @since 4.0.0
+     */
+    default int getMaxEmotes()
+    {
+        return getBoostTier().getMaxEmotes();
```

Comment:
What about the `MORE_EMOJI` feature flag?

Reply:
I don't know how you would handle that because as far as I'm aware there's no way to tell how many more emojis the guild can use

Reply:
In that case this should at least be mentioned in the documentation since the maximum returned by this might not be accurate for guilds with that feature flag.

Reply:
is `MORE_EMOJI` documented? Maybe we could request a value for the max emojis it allows.

Reply:
So I took a look into the client source code and they do it like 
`Math.max(getFeatures().contains("MORE_EMOJI")  ? EMOJI_MAX_SLOTS_MORE : EMOJI_MAX_SLOTS, getBoostTier().getMaxEmotes())` where `EMOJI_MAX_SLOTS_MORE` is 200 and `EMOJI_MAX_SLOTS` is 50